packages/wrangler/src/api/startDevWorker/LocalRuntimeController.ts

```diff
@@ -271,3 +322,46 @@ export class LocalRuntimeController extends RuntimeController {
 		this.emit("reloadComplete", data);
 	}
 }
+
+type RemoveBindingsConfigs = Partial<
+	Pick<
+		MF.ConfigBundle["bindings"],
+		| "services"
+		| "kv_namespaces"
+		| "r2_buckets"
+		| "d1_databases"
+		| "queues"
+		| "workflows"
+		| "ai"
+	>
+>;
+
+function extractRemoteBindings(
```

Comment:
This feels like it might be simpler if it operated on SDW bindings and converted afterwards? Then it would be a simple `Object.fromEntries(Object.entries(bindings).filter(([name, b]) => !!b.remote))`

Reply:
oh yeah definitely! good idea! 🙏 

(PS: the filtering function might become more complex later on depending on AI, etc..., but regardless this seems simpler/better than my current version 👍)